npm/react/package.json

```diff
@@ -21,7 +21,6 @@
     "@vitejs/plugin-react": "1.3.1",
     "axios": "0.21.2",
     "cypress": "0.0.0-development",
-    "cypress-plugin-snapshots": "1.4.4",
```

Comment:
This was actually messing things up by introducing a transitive dependency. I wrote about it [here](https://github.com/cypress-io/cypress/pull/22914#issue-1316516145) and ultimately did the change as part of this PR.